paltax/source_models.py

```diff
@@ -293,16 +294,20 @@ def modify_cosmology_params(
         Returns:
             Modified cosmology parameters.
         """
-        npz_file = np.load(self.cosmos_path)
-        images = npz_file['images']
+
+        hdf5_file = h5py.File(self.cosmos_path, 'r')
+        images = hdf5_file['galaxy_images'][:][:][:]
```

Comment:
The first [:] we'll probably want to modify to access the current batch of images. After that, I think we want to use .value (avoid the other two [:])

Reply:
I leave this up to you, but I think just using the label `images` might be more standard than specifying `galaxy_images`.